content/docs/1.2.3/deploy/install/_index.md

```diff
@@ -86,8 +86,41 @@ If MountPropagation is disabled, Base Image feature will be disabled.
 
 ### Root and Privileged Permission
 
-The longhorn manager uses `Bidirectional` mount propagation to access the host's disk (default `/var/lib/longhorn`) in the container.
-An use case is to access disk metadata and replicas directory on the host.
+The longhorn components need to access the host's disk in the container.
```

Comment:
```suggestion
Longhorn components require root access with privileged permissions to achieve volume operations and management, because Longhorn relies on system resources management on the host across different namespaces, for example, Longhorn uses `nsenter` to understand block devices' usage or encrypt/decrypt volumes on the host.
```